lib/services/show.ts

```diff
@@ -66,7 +65,8 @@ export const saveShow = async (show: PodcastIndexShowType) => {
     throw new DatabaseError(error);
   }
 
-  await saveShowToImported(data.id, show.spotifyId, show.podcastGuid);
+  // maybe we can keep it for now?
```

Comment:
**suggestion:** I guess it's better to keep this because we can import shows via our Spotify accounts to have some data.

Reply:
we probably need to have something like:

### importing via search 
we have the search result from podcast index via exact match
- check if there's an `already imported` show with the same `guid`
- if yes, navigate to show page
- if not, import and then navigate

### importing via spotify 
we have the user's spotify shows in the `onboarding/start` page (maybe move this to `/shows/import` or something)
- check if there's an `already imported` resource with the `spotify_id`
- if yes, skip
- if not, map spotify show to `open-index`
- check if there's an `already imported` show with the same `guid`
- if yes, skip
- if not, import